### PR TITLE
complete task 2 and setup test suit for next task

### PR DESCRIPTION
**Pull request by winniekwoknorth**
error handling for api/path_not_found to be done in task 3

### CODE REVIEW

File: __tests__/2-get-api-topics.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+describe("get/api/topics", () => {
+    test("get 200, then an array with all topics", () => {
+        return request(app).get("/api/topics").expect(200)
+            .then((res) => {
+                expect(res.body.topics.length).toBe(3)
+                res.body.topics.forEach((topic) => {
+                    expect(topic).toHaveProperty('slug')
+                    expect(topic).toHaveProperty('description')
+                })
```
Comment (aloush-dev) [inline __tests__/2-get-api-topics.test.js:18]: awesome job on checking the length!

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const {getTopics}= require('./controllers/topics.controllers.js')
+const { handleCustomErrors, handlePsqlErrors, handleServerErrors } = require('./err.js');
+
+app.use(express.json());
```
Comment (aloush-dev) [inline app.js:6]: Please remove the express.json(), you are not using it yet as we're not currently dealing with any incoming data.

File: models/topics.models.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const db = require('../db/connection.js')
+exports.selectTopics= () => {
+    let queryString = `SELECT * FROM topics`
```
Comment (aloush-dev) [inline models/topics.models.js:3]: Do you ever reassign this, is let the best choice?

**Review by aloush-dev — APPROVED** [review]
Awesome job!
Merge when ready.

File: models/topics.models.js (hunk 3 above)
Comment (winniekwoknorth) [reply models/topics.models.js:3]: changed to const queryString on line3

File: app.js (hunk 2 above)
Comment (winniekwoknorth) [reply app.js:6]: deleted line 6

**Pull request merged**